app/controllers/admin/editions_controller.rb

```diff
@@ -467,6 +468,16 @@ def clean_edition_parameters
     edition_params[:change_note] = nil if edition_params[:minor_change] == "true"
```

Comment:
Any reason why you've introduced a new method rather than extending the `clean_edition_parameters` method?

It seems like it'd be a good fit given the fact that it already handles `external_url` and `change_note`.

```suggestion
    edition_params[:change_note] = nil if edition_params[:minor_change] == "true"
    if edition_params[:previously_published] == "false"
      edition_params["first_published_at(1i)"] = ""
      edition_params["first_published_at(2i)"] = ""
      edition_params["first_published_at(3i)"] = ""
      edition_params["first_published_at(4i)"] = ""
      edition_params["first_published_at(5i)"] = ""
    end
```

Reply:
yea good shout. i think there's a few param cleaning methods so i sort of just followed the pattern, but they're more complex so it sort of makes sense they're discreet.